recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "6.6.2" %}
+{% set version = "6.6.3" %}
 
 package:
   name: jupyter_console
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/j/jupyter_console/jupyter_console-{{ version }}.tar.gz
-  sha256: 7385dfed8a01fd51e3c98449dae24f38c1ba015d2073835e785671653a0967fc
+  sha256: 566a4bf31c87adbfadf22cdf846e3069b59a71ed5da71d6ba4d8aaad14a53539
 
 build:
   number: 0
-  # jupyter_core does not have support for python 3.7
-  skip: true  # [py<38]
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  # Upstream supports python 3.7 but would need to use jupyter_core dependency v 4.12 exactly, which we don't have in the channel
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
```

Comment:
so according to our latest discussion the "--no-build-isolation" is must have for all packages ?

Reply:
As far as I understood it is now worth adding this flag with the latest pip we use. Here though, it probably does not make any effect.